src/wfcomman.c

```diff
@@ -1021,52 +1021,61 @@ AppCommandProc(register DWORD id)
       break;
 
    case IDM_STARTCMDSHELL:
-	   {
-		   BOOL bRunAs;
-		   BOOL bUseCmd;
-		   BOOL bDir;
-		   DWORD cchEnv;
-		   TCHAR szToRun[MAXPATHLEN];
-		   LPTSTR szDir;
+   {
+      BOOL bRunAs;
+      BOOL bUseCmd;
+      BOOL bDir;
+      DWORD cchEnv;
+      TCHAR szToRun[MAXPATHLEN];
+      LPTSTR szDir;
 
 #define ConEmuParamFormat TEXT(" -Single -Dir \"%s\"")
-           TCHAR szParams[MAXPATHLEN + COUNTOF(ConEmuParamFormat)];
+      TCHAR szParams[MAXPATHLEN + COUNTOF(ConEmuParamFormat)];

```

Comment:
Should we update this to be max(COUNTOF(ConEmuParamFormat), COUNTOF(CmdParamFormat))?  The binary code won't change because ConEmu is currently larger; this is just to describe the buffer length dependency.

Reply:
I changed this just for showing the dependency